core/invoker/src/main/scala/whisk/core/container/package.scala

```diff
@@ -83,7 +83,9 @@ package object container {
         override def toString() = s"$host:$port"
     }
 
-    sealed abstract class ContainerIdentifier(val id: String)
```

Comment:
(Sorry I missed the conversation until now, not sure why.) `Container.name` was and is supposed to be a `String`. By my mistake it became an `Any` but it's really supposed to be a `String` for when you want to generate other `String`s out of it and can't live with it being optional.